app/routes/events/components/EventAdministrate/Abacard.js

```diff
@@ -25,26 +26,33 @@ class Abacard extends React.Component<*, State> {
   };
 
   handleSelect = (result: SearchResult) => {
```

Comment:
This should be `(result: ?SearchResult=>{}` since it is allowed to click enter when there is no searchResult, resulting in an empty object here..

Reply:
I changed the `handleKeyDown` logic to only do stuff if there's at least one result, `handleSelect` shouldn't be called without a result.